src/test/java/edu/hm/hafner/util/SetTest.java

```diff
@@ -0,0 +1,149 @@
+package edu.hm.hafner.util;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+
+public class SetTest {
+    HashSet<Integer> s;
+
+    @BeforeEach
+    public void setUp() {
+        s = new HashSet<Integer>();
+    }
+
+    @Test
+    void testAdd() {
+        Assertions.assertEquals(true, s.add(Integer.valueOf(9)));
```

Comment:
```suggestion
        assertEquals(true, s.add(Integer.valueOf(9)));
```
In einer Testklasse bietet es sich an, einen statischen Import für `Assertions` zu verwenden. So würde das überall wegfallen.